static/js/giphy.js

```diff
@@ -156,9 +156,9 @@ function get_popover_placement() {
 
 export function initialize() {
     $("body").on("keydown", ".giphy-gif", ui_util.convert_enter_to_click);
-    $("body").on("keydown", "#compose_giphy_logo", ui_util.convert_enter_to_click);
+    $("body").on("keydown", ".compose_giphy_logo", ui_util.convert_enter_to_click);
 
-    $("body").on("click", "#compose_giphy_logo", (e) => {
+    $("body").on("click", ".compose_giphy_logo", (e) => {
```

Comment:
I'd have done this as a new commit at the start of the branch, rather than on top of the message-editing changes; the way you've ordered things, the `message_edit` commit is buggy and then we fix the bug.